### ISSUE TITLE
[Analytics] [Campaign Landing Pages] Custom GA events removal & adoption of "action link" va-link component w/analytics

### ISSUE THREAD

**Issue by FranECross** [department-of-veterans-affairs/va.gov-cms#18345]
## Description

The 'action link' va-link component is being enhanced (and will be available soon) to provide the destination URL in its built in analytics, and when released, can be used for Campaign Landing Pages, allowing us to remove custom GA code.

### User story

**AS A PO/PM** managing the Campaign Landing Pages
**I WANT** to (where appropriate) adopt the 'action link' va-link component, its baked-in analytics, and remove the custom GA events
**SO THAT** extraneous code is removed and only the events needed will be triggered.

## Acceptance criteria

- [x] **Element with custom events:*(Hero) "Find VA and in-network health locations" link* 
  - [x] Example page:https://www.va.gov/initiatives/covid-flu
  - [x] Event name:clp-link-click
    - [x] convert to va-link? Yes
    - [x] enable DST analytics? Yes
    - [x] remove custom code? Yes

- [x] **Element with custom events:*(Why this matters to you) "Find VA and in-network health facilities" link* 
  - [x] Example page:https://www.va.gov/initiatives/covid-flu
  - [x] Event name:clp-link-click
    - [x] convert to va-link? Yes
    - [x] enable DST analytics? Yes
    - [x] remove custom code? Yes

- [x] **Element with custom events:*(Video) "Watch the SecVA Fraud Prevention message" link* 
  - [x] Example page:https://www.va.gov/initiatives/protecting-veterans-from-fraud/ (Someone removed this video by the time I finished this spreadsheet)
  - [x] Event name:clp-link-click
    - [x] convert to va-link? Yes
    - [x] enable DST analytics? Yes
    - [x] remove custom code? Yes

- [x] **Element with custom events:*(Spotlight) Example: an action link under the "Make sure you and your family..." intro text* 
  - [x] Example page:No examples in prod but this can be used for reference: https://www.va.gov/initiatives/vote/
  - [x] Event name:clp-link-click
    - [x] convert to va-link? Yes
    - [x] enable DST analytics? Yes
    - [x] remove custom code? Yes

- [x] **Element with custom events:*(Stories) Example: an action link under the "Staying safe when considering employment..." block* 
  - [x] Example page:No examples in prod but this can be used for reference: https://www.va.gov/initiatives/protecting-veterans-from-fraud/
  - [x] Event name:clp-link-click
    - [x] convert to va-link? Yes
    - [x] enable DST analytics? Yes
    - [x] remove custom code? Yes

- [x] **Element with custom events:*(Downloadable Resources) "See more fact sheets and information" link* 
  - [x] Example page:https://www.va.gov/initiatives/emergency-room-911-or-urgent-care/
  - [x] Event name:clp-link-click
    - [x] convert to va-link? Yes
    - [x] enable DST analytics? Yes
    - [x] remove custom code? Yes

- [x] **Element with custom events:*(FAQ) "Go to the Login.gov help center" link* 
  - [x] Example page:https://www.va.gov/initiatives/sign-in-securely-with-logingov/
  - [x] Event name:clp-link-click
    - [x] convert to va-link? Yes
    - [x] enable DST analytics? Yes
    - [x] remove custom code? Yes

**Comment by randimays** [comment]
Pulling in as stretch for Sprint 6.

**Comment by randimays** [comment]
@FranECross The downloadable resources links cannot be converted to `va-link` yet because they are blocked by this accessibility defect: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2817

Do we want a separate ticket to track when this defect is complete, and then remove the custom events / convert to va-link?

**Comment by randimays** [comment]
Re: the comment to Fran above

I cut a new ticket for us to convert download links to `va-link` once the DST defect is completed and released: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18415

Otherwise, I have validated these changes successfully in production. I'm closing this ticket as complete.
